R/opendp/configure.win

```diff
@@ -0,0 +1,47 @@
+echo "***** CONFIGURE WIN *****"
```

Comment:
Two factors here:
1. parameterizing it seems like it would add a fair amount of complexity
2. following R conventions

I think neither factor is conclusive, so I'm flexible to changing it. Although these files are extremely touchy.